docs/documentation/tasks/rails/index.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Custom Rails Tasks
+layout: default
+---
+
+Many of these tasks probably require [Capistrano::Rails](https://github.com/capistrano/rails).
+
+### Run arbitrary rake tasks from environment variables
+
+From [Capistrano/Rails PR #209](https://github.com/capistrano/rails/pull/209)
+
+```ruby
+namespace :ruby do
+  desc 'Runs any rake task, cap deploy:rake task=db:rollback'
```

Comment:
Should this namespace be `:ruby` or `:deploy`?

I like to encourage people to avoid the `:deploy` namespace because that way you can tell what tasks come out of the box vs those you've added yourself.